configs/body3d/simple_baseline/h36m/linear_h36m_block2x1024_maxnorm.py

```diff
@@ -0,0 +1,136 @@
+log_level = 'INFO'
+load_from = None
+resume_from = None
+dist_params = dict(backend='nccl')
+workflow = [('train', 1)]
+checkpoint_config = dict(interval=1)
+evaluation = dict(interval=1, metric='mpjpe', key_indicator='MPJPE')
+
+# optimizer settings
+optimizer = dict(
+    type='Adam',
+    lr=1e-3,
+)
+optimizer_config = dict(grad_clip=None)
+# learning policy
+lr_config = dict(
+    policy='step',
+    by_epoch=False,
+    step=100000,
+    gamma=0.96,
+)
+
+total_epochs = 200
+
+log_config = dict(
+    interval=50,
+    hooks=[
+        dict(type='TextLoggerHook'),
+        # dict(type='TensorboardLoggerHook')
+    ])
+
+channel_cfg = dict(
+    num_output_channels=17,
+    dataset_joints=17,
+    dataset_channel=[
+        [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16],
+    ],
+    inference_channel=[
+        0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16
+    ])
+
+# model settings
+model = dict(
+    type='PoseLifter',
+    pretrained=None,
+    backbone=dict(
+        type='TCN',
+        in_channels=2 * 16,
+        stem_channels=1024,
+        num_blocks=2,
+        kernel_sizes=(1, 1, 1),
+        dropout=0.5,
+        max_norm=1.0),
+    keypoint_head=dict(
+        type='TemporalRegressionHead',
+        in_channels=1024,
+        num_joints=16,
+        max_norm=1.0,
+        loss_keypoint=dict(type='MPJPELoss')),
```

Comment:
The original TensorFlow implementation uses MSELoss instead of MPJPELoss. Maybe this has some effect on training?